sqlite-db/src/rollover/overwrite.rs

```diff
@@ -16,64 +15,47 @@ use sqlx::Connection as SqlxConnection;
 use sqlx::Sqlite;
 use sqlx::Transaction;
 
-pub async fn insert(
+mod delete;
+
+/// Overwrite a CFD's latest rollover data.
+///
+/// After a successful rollover, we can forget about the previous `Dlc`, `FundingFee` and
+/// `CompleteFee`.
+pub async fn overwrite(
```

Comment:
Do we need to think about mutual exclusion here?

Reply:
This was discussed offline and I came to the conclusion that we are already implementing mutual exclusion when writing to the database because we only have one call to `append_event` in production:

https://github.com/itchysats/itchysats/blob/1305340f1fbb8567ed72f594bbf03013cbae93da/daemon/src/process_manager.rs#L74

I'm gonna take this conversation to a separate ticket because it will likely get lost here.

Reply:
Here it is: https://github.com/itchysats/itchysats/issues/2663.